phonebook/main.js

```diff
@@ -2,7 +2,7 @@
 
 const http = require('http')
 let contacts = [
-    {id: 0, first: 'Dylan', last: 'Bialy', email: 'dylanbailey@gmail.com'},
```

Comment:
Like that you made this visually more appealing by adding a space between brackets and letters.